x-pack/filebeat/module/infoblox/nios/test/generated.log-expected.json

```diff
@@ -0,0 +1,2476 @@
+        "observer.vendor": "Infoblox",
+        "rsa.internal.data": "eriame",
+        "rsa.internal.event_desc": "lorema",
+        "rsa.internal.messageid": "rc6",
+        "rsa.misc.event_source": "enim2780.www.lan",
+        "rsa.time.day": "20",
+        "rsa.time.event_time": "2020-06-20T06:35:42.000Z",
+        "rsa.time.month": "June",
+        "service.type": "infoblox",
+        "tags": [
+            "infoblox.nios",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "watchdog",
+        "event.dataset": "infoblox.nios",
+        "event.module": "infoblox",
+        "event.original": "July 4 11:38:16 emporinc5075.internal.host watchdog[atcu]: oremagna could not be opened, errno = ationu",
+        "file.name": "oremagna",
+        "fileset.name": "nios",
+        "input.type": "log",
+        "log.offset": 1124,
+        "observer.product": "Network",
+        "observer.type": "IPAM",
+        "observer.vendor": "Infoblox",
+        "rsa.internal.data": "atcu",
+        "rsa.internal.messageid": "watchdog",
+        "rsa.misc.event_source": "emporinc5075.internal.host",
+        "rsa.misc.result_code": "ationu",
+        "rsa.time.day": "4",
+        "rsa.time.event_time": "2020-07-04T13:38:16.000Z",
+        "rsa.time.month": "July",
+        "service.type": "infoblox",
+        "tags": [
+            "infoblox.nios",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "shutdown",
+        "event.dataset": "infoblox.nios",
+        "event.module": "infoblox",
+        "event.original": "July 18 18:40:50 strude910.internal.local 10.27.72.147 shutdown: shutting down for system reboot",
+        "fileset.name": "nios",
+        "input.type": "log",
+        "log.offset": 1228,
+        "observer.product": "Network",
+        "observer.type": "IPAM",
+        "observer.vendor": "Infoblox",
+        "rsa.internal.event_desc": "shutting down for system reboot",
+        "rsa.internal.messageid": "shutdown",
+        "rsa.misc.event_source": "strude910.internal.local",
+        "rsa.time.day": "18",
+        "rsa.time.event_time": "2019-07-18T20:40:50.000Z",
+        "rsa.time.month": "July",
+        "service.type": "infoblox",
+        "tags": [
+            "infoblox.nios",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "ntpd",
+        "event.dataset": "infoblox.nios",
+        "event.module": "infoblox",
+        "event.original": "August 2 01:43:25 fugit7668.www5.invalid -ntpd_initres: ntpd exiting on signal 15",
+        "fileset.name": "nios",
+        "input.type": "log",
+        "log.offset": 1325,
+        "observer.product": "Network",
+        "observer.type": "IPAM",
+        "observer.vendor": "Infoblox",
+        "rsa.internal.event_desc": "exiting",
+        "rsa.internal.messageid": "ntpd",
+        "rsa.misc.event_source": "fugit7668.www5.invalid",
+        "rsa.time.day": "2",
+        "rsa.time.event_time": "2019-08-02T03:43:25.000Z",
```

Comment:
this should probably be the value for `@timestamp` no?

Reply:
The problem with this logs is that they don't contain a year, only day/month.

In that case I set the year to the current year (or the past year, if that would put the date more than 2 days in the future).

Due to this, I have to configure the tests to drop `@timestamp` for this fileset (in `test_modules.py`).